psi4/src/psi4/cc/cclambda/overlap.cc

```diff
@@ -32,6 +32,7 @@
 */
 #include <cstdio>
 #include "psi4/libdpd/dpd.h"
+#include "psi4/libmints/wavefunction.h"
```

Comment:
It's not obvious to me why the header is needed, given the only diff in the file is that the `overlap` function now returns a `double`.

Reply:
Artifact of a previous draft that I forgot to remove.